src/Features/Core/Portable/Remote/RemoteArguments.cs

```diff
@@ -114,8 +115,12 @@ private class NavigateToSearchResult : INavigateToSearchResult
         }
     }
 
+    /// <summary>
+    /// Note: this is intentionally a class, not struct, to avoid hitting .NET Framework loader bug
+    /// that fails to load a struct S declaring a field of type ImmutableArray of S.
```

Comment:
=-o

is there a bug id for that?

how have we not constantly been hitting that? :)

Reply:
oh... it has to `struct S { ImmutableArray<S> ...}`  got it.

Reply:
Yes, there is a bug that's closed "won't fix" since we don't fix anything on .NET Framework.